src/support/thread_group.c

```diff
@@ -23,6 +23,7 @@ __thread_run(void *arg)
 	session = thread->session;
 
 	for (;;) {
+		WT_ERR(WT_SESSION_CHECK_PANIC(session));
```

Comment:
If the system panics, I don't see anything that will break this loop.  However, I'm happy to let that be considered under WT-3245 with other threads too.